transport.rb

```diff
@@ -0,0 +1,45 @@
+class Transport
+
+  include Comparable , Enumerable
+
+  attr_accessor :current_weight , :available
```

Comment:
It provides setters too. Propose to leave only getters

Reply:
Done